src/server/validations/products.validation.js

```diff
@@ -0,0 +1,14 @@
+import * as yup from 'yup';
+
+
+export const schemaId = yup.object().shape({
+    id: yup.number().positive().integer()
+});
+export const schemaProductEdit = yup.object().shape({
+    title: yup.string().min(2),
```

Comment:
You need to validate max length here, since the title column in the database have maximum size of 255 characters.